### ISSUE TITLE
Passing project path causes tox to fail.

### ISSUE THREAD

**Issue by terjekv** [The-Compiler/vscode-python-tox#31]
## Bug description

Running tox adds the project path as an argument without prefixing it with --, which tox 4.2.3 (at least) doesn't like.

## To Reproduce

(env) ➜  testproj main ✓$ tox -e format [...]/projects/private/testproj
usage: tox [-h] [--colored {yes,no}] [-v | -q] [--exit-and-dump-after seconds] [-c file] [--workdir dir] [--root dir] [--runner {virtualenv}] [--version] [--no-provision [REQ_JSON]] [--no-recreate-provision] [-r] [-x OVERRIDE]
           {run,r,run-parallel,p,depends,de,list,l,devenv,d,config,c,quickstart,q,exec,e,legacy,le} ...
tox: error: unrecognized arguments: [...]/projects/private/testproj
hint: if you tried to pass arguments to a command use -- to separate them from tox ones

## Expected behavior

A working tox. :)

## Additional context

Using `tox -e format -- [...]/projects/private/testproj` works.

**Comment by The-Compiler** [comment]
I don't quite follow - you are running tox from the extension and passing the project path as additional arguments in the VSCode UI?

**Comment by terjekv** [comment]
I'm running Tox from the extension, either via the "play" buttons in the "testing" interface from the main side bar, or from hitting the "play" button in the tox file itself.

**Comment by The-Compiler** [comment]
Alright, but where is the path coming from? Because it's not the extension adding it...

**Comment by terjekv** [comment]
Huh. That is a good question. I have installed the extension and done no configuration at all. The path is the path of the project so it's getting something "correct" from somewhere.

**Comment by The-Compiler** [comment]
Ah! I can indeed reproduce when using it that way, personally I've ever only used the commands via the command palette (or keybindings in my case).

When running tox from the tests interface, It looks like the directory is indeed added here:

https://github.com/The-Compiler/vscode-python-tox/blob/63176541748b4101aad490b6f3dcda8a30602df4/src/testController.ts#L41

I believe this is due to 0b558fa81bf5989c1d83ab084012c37159584f93 (which added the testing integration) not accounting for the changes in b6a92efb082b8db582d1350fa449ac101da4551e (which adds a second argument `toxArguments` to `runTox`). Thus, `cwd` now gets passed as `toxArguments` instead of `projDir`.

@starlord-daniel if you have the bandwidth to confirm that I'm on the right track there, and/or to open a PR with a fix, that'd be most appreciated! If not, I'm going to go ahead with a fix once I find the time to work on this extension again, but it might be a bit, as I'm very busy until March again.

**Comment by terjekv** [comment]
Ah, thanks for the digging. Good to know I wasn't going insane. :)

**Comment by starlord-daniel** [comment]
@The-Compiler Yes, that seems to be the issue. Unfortunately I don't have any bandwidth right now. Happy to review though.
